ddtrace/profiling/exporter/pprof.pyx

```diff
@@ -74,7 +74,9 @@ class _PprofConverter(object):
 
     # A dict where key is a (Location, [Labels]) and value is a a dict.
     # This dict has sample-type (e.g. "cpu-time") as key and the numeric value.
-    _location_values = attr.ib(factory=lambda: collections.defaultdict(dict), init=False, repr=False)
+    _location_values = attr.ib(
+        factory=lambda: collections.defaultdict(lambda: collections.defaultdict(lambda: 0)), init=False, repr=False
```

Comment:
is `collections.defaultdict(int)` more clear here?

Reply:
Since I didn't know that `int()` was returning 0, I'm not sure 😅 
I learned something!